src/tests/Interop/UnmanagedCallersOnly/UnmanagedCallersOnlyTest.cs

```diff
@@ -371,7 +379,7 @@ .locals init ([0] native int ptr)
         il.Emit(OpCodes.Nop);
```

Comment:
The dynamic IL generation can be moved to the IL project too. It would improve the test coverage for AOT.

It can be done as a separate PR.